tests/unittests/test_http_functions.py

```diff
@@ -98,34 +97,22 @@ def check_log_async_logging(self, host_out: typing.List[str]):
         self.assertIn('hello info', host_out)
         self.assertIn('and another error', host_out)
 
-    @unittest.skip("Reverting the debug logs PR as host currently cannot handle"
-                   "apps with lot of debug statements. Reverting PR: "
-                   "azure-functions-python-worker/pull/745")
     def test_debug_logging(self):
```

Comment:
Do we need all of these tests if you are already testing this in the test_enable_debug_logging_functions.py? If not, then I guess these can be removed

Reply:
yes, this is kept for testing flag unset case